source/blog/2019-02-22-the-ember-times-issue-86.md

```diff
@@ -0,0 +1,128 @@
+---
+title: The Ember Times - Issue No. 86
+author: Chris Ng, Gaurav Munjal, Amy Lam, Jessica Jordan, the crowd
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
+alias : "blog/2019/02/22-the-ember-times-issue-86.html"
+responsive: true
+---
+
+<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+
+Check out the two follow up RFCs to make Tracked Properties better  👣, an Ember community experience 🤝...
+
+...and an exclusive interview with Ember contributor @cibernox 💬 !
+
+---
+
+## [SECTION TITLE](#section-url)
+
+
+---
+
+## [SECTION TITLE](#section-url)
+
+
+---
+
+## [SECTION TITLE](#section-url)
+
+
+---
+
+## RFCs for [Decorator Support](https://github.com/emberjs/rfcs/pull/440) and [Tracked Property Classic Class Descriptor](https://github.com/emberjs/rfcs/pull/442) 👣
```

Comment:
How come this doesn't appear on the deployment rn? Because it's a WIP PR? Writing a comment to check this locally (header with 2 links) later, we can do one link `/rfcs` for Goodbits

Reply:
I think the post doesn't show up in the app we currently use for the blog if the date is in the future. It should show up today though or you can check it out by running the app locally and setting your computer's time settings into the future or check it out today:
https://ember-website-staging-pr-3821.herokuapp.com/blog/2019/02/22/the-ember-times-issue-86.html

Feel free to let me know if I missed something about the question and yes I agree with what you suggested about the header title - I can update it in a copy-edit :+1:

Reply:
Oh I see, I thought it was something with `WIP` PR.

And yes I meant two links in one header wouldn't work for Goodbits.

Thank you!